spec/integration/rails/integration_spec.rb

```diff
@@ -19,6 +19,11 @@
     expect(Hashie.logger).to eq(Rails.logger)
   end
 
+  it 'works with #except' do
+    mash = Hashie::Mash.new(x: 1, y: 2)
+    expect(mash.except(:x)).to be_a(Hashie::Mash)
```

Comment:
Let's verify the behavior that we want. We want the keys to be correct, so let's verify that.

```suggestion
    expect(mash.except(:x)).to eq Hashie::Mash.new(y: 2)
    expect(mash.except('x')).to eq Hashie::Mash.new(y: 2)
```

Reply:
Good call. Testing this, the case where we use `except(:symbol)` fails, and there's not a workaround we can do while only implementing standard-library methods. 

I still think there's value in adding slice to Mash. 
We would have to call out the fact that it needs to be used with string keys and not symbols when used in Rails without a railtie.

Then a railtie could be created to properly handle the usage of `except`.